packages/zwave-js/src/lib/commandclass/WakeUpCC.ts

```diff
@@ -237,7 +245,8 @@ export class WakeUpCC extends CommandClass {
 default wakeup interval: ${wakeupCaps.defaultWakeUpInterval} seconds
 minimum wakeup interval: ${wakeupCaps.minWakeUpInterval} seconds
 maximum wakeup interval: ${wakeupCaps.maxWakeUpInterval} seconds
-wakeup interval steps:   ${wakeupCaps.wakeUpIntervalSteps} seconds`;
+wakeup interval steps:   ${wakeupCaps.wakeUpIntervalSteps} seconds
+wake up on demand supported: ${wakeupCaps.wakeUpOnDemandSupported}`;
```

Comment:
```suggestion
wakeup on demand supported: ${wakeupCaps.wakeUpOnDemandSupported}`;
```
for consistency. Or make it `wake up` everywhere else :)

Reply:
I standardized on "wake up", because that's how it was written in the spec sheet. However, it's definitely more important to be consistent within zwave-js, and changing existing conventions without good reasons is dumb... so "wakeup" (one word) it is.

Reply:
Just looked at the existing code, and it looks like "wake up" is spelled:

1. "wakeup" in the interview logs
2. "Wake Up" in the `@ccValueMetaData` label
3. "wake-up" in the `Set` logs.

🤣 

I'm still changing it to "wakeup" here for local consistency.